x/incentive/params.go

```diff
@@ -11,13 +11,10 @@ const secondsPerDay = 86400
 // DefaultParams returns a default set of parameters.
 func DefaultParams() Params {
 	return Params{
-		MaxUnbondings:           5,
-		UnbondingDurationLong:   secondsPerDay * 14,
-		UnbondingDurationMiddle: secondsPerDay * 7,
-		UnbondingDurationShort:  secondsPerDay,
-		TierWeightMiddle:        sdk.MustNewDecFromStr("0.8"),
-		TierWeightShort:         sdk.MustNewDecFromStr("0.5"),
-		CommunityFundAddress:    "",
+		MaxUnbondings:        5,
+		UnbondingDuration:    secondsPerDay * 1,
+		EmergencyUnbondFee:   sdk.MustNewDecFromStr("0.01"),
+		CommunityFundAddress: "",
```

Comment:
Seems more efficient to keep it with params (since it behaves like one) than to create a separate gov message (not `MsgSetParams`) to manage this address

Reply:
Fixed